translate/cloud-client/translate_v3_translate_text_with_model.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# DO NOT EDIT! This is a generated sample ("Request",  "translate_v3_translate_text_with_model")
+
+# To install the latest published package dependency, execute the following:
+#   pip install google-cloud-translate
+
+# sample-metadata
+#   title: Translating Text with Model
+#   description: Translating Text with Model
+#   usage: python3 translate_v3_translate_text_with_model.py [--model_id "[MODEL ID]"] [--text "Hello, world!"] [--target_language fr] [--source_language en] [--project_id "[Google Cloud Project ID]"] [--location global]
+
+# [START translate_v3_translate_text_with_model]
+from google.cloud import translate_v3
+
+
+def sample_translate_text_with_model(
+    model_id, text, target_language, source_language, project_id, location
+):
+    """
+    Translating Text with Model
+
+    Args:
+      model_id The `model` type requested for this translation.
+      text The content to translate in string format
+      target_language Required. The BCP-47 language code to use for translation.
+      source_language Optional. The BCP-47 language code of the input text.
+    """
+
+    client = translate_v3.TranslationServiceClient()
+
+    # TODO(developer): Uncomment and set the following variables
+    # model_id = '[MODEL ID]'
+    # text = 'Hello, world!'
+    # target_language = 'fr'
+    # source_language = 'en'
+    # project_id = '[Google Cloud Project ID]'
+    # location = 'global'
+    contents = [text]
+    parent = client.location_path(project_id, location)
+    model_path = 'projects/{}/locations/{}/models/{}'.format(project_id, 'us-central1', model_id)
```

Comment:
Is there not a python client method?
`client.model_path(project_id, 'us-central1', model_id)`

If not, that's a bummer and we should ask for one.

Reply:
it exists in automl client, which is presumably where the model will be from.

Reply:
Ah, but we probably shouldn't do an import just for that.